SAMv1.tex

```diff
@@ -224,6 +224,11 @@ \subsection{The header section}
     \emph{Valid values}: {\tt none} (default), {\tt query} (alignments are
     grouped by {\sf QNAME}), and {\tt reference} (alignments are grouped by
     {\sf RNAME}/{\sf POS}).\\\cline{1-3}
+  & {\tt SS} & Sub-Sorting order of alignments. Valid values are of the form {\tt <sort-order>:<sub-sort>}, where {\tt <sort-order>} is the same value stored in the {\tt SO} attribute and {\tt <sub-sort>} is an implementation-dependent string further describing the sort order.
+    E.g. if an algorithm relies on a {\tt coordinate} sort that, at each coordinate, is further sorted by queryname the header could contain {\tt @HD SO:coordinate SS:coordinate:queryname}.
+    If the primary sort is not one of the predefined primary sort orders, then {\tt unsorted} should be used, and the subsort is effectively the major sort.
+    E.g. if sorted by an auxillary tag {\tt MI} then by coordinate then the header could contain {\tt @HD SO:unsorted SS:unsorted:MI:coordinate}.
+    Values must match be colon separated alphanumerics with underscores and hyphens permitted, i.e. matching the regular expression {\tt [A-Za-z0-9:\_-]+}. \\\cline{1-3}
```

Comment:
` Values must match be colon` - I think there's a word missing here

Reply:
Also, are colons allowed to be in the name of the sub-sort?  Or is a colon the delimiter for multiple sub-sorts?  Or is the example `SS:unsorted:MI:coordinate` interpreted as the `<sort-order>` is `unsorted`, and the `<sub-sort>` is `MI:coordinate`, where the latter can be interpreted independently?

Reply:
I fixed the "must match be colon" mis-edit.

Regarding colons in the name, no they are not allowed and I think this should be clear enough already by "colon separated".  By definition, any colon in a name is considered to be a separator so they cannot exist.